client/tests/integration/events/pipeline.rs

```diff
@@ -8,22 +8,26 @@ use test_network::*;
 
 use super::Configuration;
 
+#[allow(dead_code)]
```

Comment:
So the dead code shouldn't be removed, given that it should also be possible to re-enable these tests without re-compiling.

Reply:
```suggestion
// Needed to re-enable ignored tests. 
#[allow(dead_code)]
```